pages/docs/opentelemetry/get-started.mdx

```diff
@@ -238,23 +238,41 @@ In addition, we map many GenAI specific properties to properties in the Langfuse
 First and foremost, we stick to the [OpenTelemetry Gen AI Conventions](https://opentelemetry.io/docs/specs/semconv/attributes-registry/gen-ai/), but also map vendor specific properties from common frameworks.
 All attributes and resourceAttributes are available within the Langfuse `metadata` property as a fallback.
 
-Below, we share a non-exhaustive list of mappings that Langfuse applies:
+Langfuse uses the `langfuse.*` namespace to map properties from OpenTelemetry to the Langfuse data model.
```

Comment:
In this section to we need to be specific about the fact that these are "Span Attributes"?

Reply:
In the OTel context attributes are clearly linked to spans from my understanding. The only other option are resourceAttributes and one would mention them explicitly if they are meant. So I think it should work like it is.

Reply:
sounds good